srtcore/sync_posix.cpp

```diff
@@ -291,10 +291,24 @@ void Condition::init()
 #if SRT_SYNC_CLOCK == SRT_SYNC_CLOCK_GETTIME_MONOTONIC
     pthread_condattr_t  CondAttribs;
     pthread_condattr_init(&CondAttribs);
-    pthread_condattr_setclock(&CondAttribs, CLOCK_MONOTONIC);
-    attr = &CondAttribs;
+    if (pthread_condattr_setclock(&CondAttribs, CLOCK_MONOTONIC) != 0)
+    {
+        pthread_condattr_destroy(&CondAttribs);
+        LOGC(inlog.Warn, log << "pthread_condattr_setclock failed!");
```

Comment:
Note that if that call failed, it means that we don't have the monotonic clock set up for the CV. I think such a thing should be reported as Fatal (not Warn).
```suggestion
        LOGC(inlog.Fatal, log << "IPE: pthread_condattr_setclock failed to set up a monotonic clock for a CV");
```